davisinteractive/utils/mask_test.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from PIL import Image
+import numpy as np
+
+from .mask import combine_masks
+
+
```

Comment:
Good tests using some raw data to test it!

My only suggestion is to define here:
```python
TEST_DIR = os.path.join(os.path.dirname(__file__), 'masks_test')
```

And the resulting lines on the tests won't be so nested.

Reply:
Done!